dask/dataframe/tests/test_dataframe.py

```diff
@@ -172,6 +172,20 @@ def test_Index():
         pytest.raises(AttributeError, lambda: ddf.index.index)
 
 
+def test_axes():
+    pdf = pd.DataFrame({"col1": [1, 2], "col2": [3, 4]})
+    df = dd.from_pandas(pdf, npartitions=2)
+    assert len(df) == 2
```

Comment:
Similar comment here

```suggestion
    assert len(df.axes) == len(pdf.axes)
```

Reply:
Adjusted in https://github.com/dask/dask/pull/8069/commits/6dd6fafad932c4d5383afa60cfe6dee324123668